draft-tigress-requirements.md

```diff
@@ -78,8 +78,9 @@ Tigress:
 
 --- abstract
 
-This document describes the use cases necessitating the secure transfer of digital credentials. The document also comprises a proposal, and defines requirements and scope.
-
+This document describes the use cases necessitating the secure transfer of digital credentials between two devices. 
+Secure credentials may represent a digital key to a hotel room, a digital key to a door lock in a house or a digital key to a car. 
+The document also comprises a proposal, and defines requirements and scope.
```

Comment:
@dimmyvi: I really like the new sentence in line 82, but unfortunately the sentence in line 83 seems to be lost a bit now.
Maybe we can add line 83 to line 81, e. g.:
"This document describes the use cases necessitating the secure transfer of digital credentials between two devices and defines general assumptions, requirements and the scope of the corresponding Internet-draft."

Reply:
Done.